tests/pages/bigtest/package-show.js

```diff
@@ -0,0 +1,23 @@
+import {
+  clickable,
+  isPresent,
+  page,
+  property,
+} from '@bigtest/interaction';
+
+@page class PackageShowModalPage {
```

Comment:
I think we can safely call this `PackageShowModal` now that we don't have to have a named export.